src/vma/dev/ring_eth_cb.cpp

```diff
@@ -252,7 +257,8 @@ int ring_eth_cb::allocate_umr_mem()
 			// network not accessible to application
 			m_p_mem_rep_list[index].base_addr = base_ptr;
 			m_p_mem_rep_list[index].byte_count[0] = net_len;
-			m_p_mem_rep_list[index].stride[0] = net_len;
+			// optimize write header to the same place
```

Comment:
Consider to change comment to "optimize write header to the same physical address"